pkgs/racket-doc/syntax/scribblings/format.scrbl

```diff
@@ -0,0 +1,106 @@
+#lang scribble/manual
+
+@(require scribble/example
+          "common.rkt"
+          (for-label racket/function
+                     racket/format
+                     syntax/format
+                     syntax/srcloc))
+
+@(define make-format-eval (make-eval-factory '[syntax/format]))
+
+@title[#:tag "format"]{Creating Formatted Identifiers}
+@defmodule[syntax/format]
+
+@defproc[(~id [v (or/c identifier? string? symbol? keyword? char? number?)] ...
```

Comment:
A `v` can be an `identifier?` too

Reply:
I don’t understand your comment… `identifier?` is already the first predicate in the `or/c`.

Reply:
Oops